lib/miq_automation_engine/service_models/miq_ae_service_auth_private_key.rb

```diff
@@ -1,4 +1,5 @@
 module MiqAeMethodService
   class MiqAeServiceAuthPrivateKey < MiqAeServiceAuthentication
+    expose :auth_key
```

Comment:
Do not expose this key.  From the `ContainerDeployment` object it can be referenced by accessing the active record AuthPrivateKey model.